### PR TITLE
Add button to sales dashboard to update Chargebee cache

### PR DESCRIPTION
**Pull request by matthewelwell**
## Changes

Add button to Sales Dashboard to update Chargebee cache

## How did you test this code?

Added a unit test and ran the application manually to verify the button calls the endpoint correctly.

### CODE REVIEW

File: api/organisations/chargebee/tasks.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from organisations.chargebee.cache import ChargebeeCache
+from task_processor.decorators import register_task_handler
+
+
+@register_task_handler()
+def update_chargebee_cache():
+    chargebee_cache = ChargebeeCache()
+    chargebee_cache.refresh()
```
Comment (khvn26) [inline api/organisations/chargebee/tasks.py:8]: Two questions:

1. How is the cache not being updated in the first place? Some webhook getting skipped by Chargebee? Any errors on our side?
2. Can/should this also be a cron job?

**Review by khvn26 — APPROVED** [review]

File: api/organisations/chargebee/tasks.py (hunk 1 above)
Comment (matthewelwell) [reply api/organisations/chargebee/tasks.py:8]: It sets the cache with a 12h expiry then automatically refreshes it the next time data from the cache is requested. This was a quick and dirty solution to allow us to clear it if needs be when a new plan / add-on is added to chargebee, without having to handle a new webhook (which is obviously the correct solution).

Comment (khvn26) [reply api/organisations/chargebee/tasks.py:8]: Understood. Want to create the webhook issue while you're at it?

Comment (matthewelwell) [reply api/organisations/chargebee/tasks.py:8]: Done https://github.com/Flagsmith/flagsmith/issues/2246

**Pull request merged**